app/src/main/java/com/example/capstoneproject/MainActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.capstoneproject
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.example.capstoneproject.ui.main.MainFragment
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.main_activity)
+        /*if (savedInstanceState == null) {
```

Comment:
I suggest you remove unused code and commented code to avoid confusing other developers :D